hw3/01-characters.js

```diff
@@ -0,0 +1,50 @@
+// url for the Thrones API
+const url = 'https://thronesapi.com/api/v2/Characters';
+
+
+let app = document.querySelector('#results');
+const loadingImg = document.getElementById('loading');
+const render = document.createElement('div');
+render.classList.add(
+	'container',
+	'd-flex',
+	'flex-wrap',
+	'justify-content-center'
+);
+app.appendChild(render);
+const fetchData = async (url) => {
+	// Add your code here
+	try {
+		const response = await fetch(url);
+        console.log("RESPONSE", response)
+		const data = await response.json();
+        
+		data.forEach((character) => {
```

Comment:
Everything works! I'd just recommend splitting the function to fetch the Game of Thrones data with the logic behind parsing that data and displaying the images. Although for this particular assignment it doesn't really matter, getting the habit of modularizing your code and keeping separations of concern in mind.

Reply:
It works as expected. A small suggestion will be to split you code in the try block .